### PR TITLE
feat: finish enable notifications

### LINKED ISSUE #17506: Finish the "Enable notifications" screen
# Definition of done

- [ ] 1. Update the copy on the top text container - [message](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=6117:67315&mode=design#569642626)
- [ ] 2. Add missing icon to the "Enable notifications" button - [message](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=6117:67315&mode=design#569642311)
- [ ] 3. Add animation (get in touch with the design team or @pedro-et) - [message](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=6117:67315&mode=design#569642792)

### PR DESCRIPTION
**Pull request by BalogunofAfrica**
fixes https://github.com/status-im/status-mobile/issues/17506

### Summary

This PR implements the enable notifications screen as seen in the design feedback here: https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=6117-67315&mode=design&t=DAsT60Sje05FrhhM-0#569642792

#### Platforms

- Android
- iOS

status: ready

### CODE REVIEW

File: src/status_im2/contexts/onboarding/enable_notifications/view.cljs [hunk 1]
```diff
@@ -44,16 +47,38 @@
...
       (i18n/label :t/maybe-later)]]))
 
-(defn enable-notifications
+(defn enable-notifications-parallax
+  []
+  (let [stretch (if rn/small-screen? -40 -25)]
+    [parallax/video
+     {:layers  (:notifications resources/parallax-video)
+      :stretch stretch}]))
+
```
Comment (J-Son89) [inline src/status_im2/contexts/onboarding/enable_notifications/view.cljs:62]: extra line

File: src/status_im2/navigation/screens.cljs [hunk 2]
```diff
@@ -207,7 +207,7 @@
                  :animations             (merge transitions/new-to-status-modal-animations
                                                 transitions/push-animations-for-transparent-background)
                  :modalPresentationStyle :overCurrentContext}
-     :component enable-notifications/enable-notifications}
+     :component enable-notifications/view}
```
Comment (J-Son89) [inline src/status_im2/navigation/screens.cljs:213]: 🙌

**Review by J-Son89 — APPROVED** [review]

**Comment by J-Son89** [comment]
I tested this and looks good 👌 - admittedly the app did crash on the onboarding process but on the "welcome" page and I couldn't reproduce this afterwards, only happened once out of a few trys

**Review by Francesca-G — APPROVED** [review]
Looks great ✨

**Review by ajayesivan — APPROVED** [review]
Nice work 🎉

**Comment by pavloburykh** [comment]
@BalogunofAfrica thank for the PR and sorry for delay with testing. No issues from my side. Ready for design review @Francesca-G

**Comment by BalogunofAfrica** [comment]
> @BalogunofAfrica thank for the PR and sorry for delay with testing. No issues from my side. Ready for design review @Francesca-G

Thanks @pavloburykh @Francesca-G has already reviewed

**Comment by pavloburykh** [comment]
> @Francesca-G has already reviewed

Great! Then we are ready for merge.

**Pull request merged**